.github/workflows/test_on_push.yaml

```diff
@@ -8,6 +8,16 @@ env:
   ROCK_NAME: "tracing"
 
 jobs:
+  version-check:
+    # We need this job to run only on push with tag.
+    if: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags/') }}
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check module version
+        uses: tarantool/actions/check-module-version@master
+        with:
+          module-name: 'opentracing'
```

Comment:
What's about zipkin version check?

Reply:
Done, now there are jobs `version-check-opentracing` and `version-check-zipkin`